### PR TITLE
fix:(DB\Smart_scripts):Blazing Invader should not despawn

### PR DESCRIPTION
**Pull request by Ragelezz**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7369
- Closes https://github.com/chromiecraft/chromiecraft/issues/1440

## Tests Performed:

- .go c id 14460 and look for the portal, it despawned either after 500ms (when out of combat) either 2 mins after spawned.
- 

## How to Test the Changes:

1. .go c id 14460 and look for the portal, stay near portal and look at elementals, they won't despawn anymore
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1632568180601547400.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632568180601547400');
+
+UPDATE `smart_scripts` SET `action_param3`='990000' WHERE  `entryorguid`=179666 AND `source_type`=1 AND `id`=0 AND `link`=0;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=14460;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1632568180601547400.sql:5]: miss source_type here, you need to delete per entryorguid and per source_type, so add AND source_type = 0

File: data/sql/updates/pending_db_world/rev_1632568180601547400.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632568180601547400');
+
+UPDATE `smart_scripts` SET `action_param3`='990000' WHERE  `entryorguid`=179666 AND `source_type`=1 AND `id`=0 AND `link`=0;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1632568180601547400.sql:3]: action_param3 is an unsigned int, not a string
```suggestion
UPDATE `smart_scripts` SET `action_param3`=990000 WHERE  `entryorguid`=179666 AND `source_type`=1 AND `id`=0 AND `link`=0;
```

**Comment by jamadaha** [comment]
**Tested**

They don't despawn, but rather just keep spawning
image

**Ehhh, not sure it's working as intended**

**Comment by FrancescoBorzi** [comment]
> **Tested**
> 
> They don't despawn, but rather just keep spawning image
> 
> **Ehhh, not sure it's working as intended**

thanks @jamadaha for testing this

@Ragelezz please check this

**Comment by Ragelezz** [comment]
Read comments on this PR, https://github.com/azerothcore/azerothcore-wotlk/pull/8082
Someone had the same question\issue but official posts say otherwise.

**Pull request closed without merging**